.github/workflows/rbc_test.yml

```diff
@@ -113,8 +113,8 @@ jobs:
             omniscidb-from: docker
           - os: ubuntu-latest
             python-version: 3.8
-            omniscidb-version: 5.6.4
-            docker-image: omnisci/core-os-cpu:v5.6.4
+            omniscidb-version: 5.7.0
+            docker-image: omnisci/core-os-cpu:v5.7.0
```

Comment:
If we can test omniscidb 5.7 using conda-forge, we don't need to test it again using docker.